os/arch/arm/Kconfig

```diff
@@ -189,8 +260,18 @@ config ARCH_ARMV7R_FAMILY
 	bool
 	default n
 
+config ARCH_ARMV7A_FAMILY
+	bool
+	default n
+
+config ARCH_ARMV7A
```

Comment:
Why is this config still present? Let us only use the FAMILY config. This will be a duplicate.